src/bootstrap-customisations/_alerts.scss

```diff
@@ -0,0 +1,35 @@
+.alert {
+    border-color: transparent;
+    color: $body-color;
+    display: flex;
+    font-size: $fhi-font-size-3;
+    line-height: 1.35;
+
+    [class*="icon-"] {
+        background-size: contain;
+        flex-shrink: 0;
+        margin-right: $fhi-space-2;
+    }
+}
+
+.alert-dismissible .btn-close {
+    top: -1px;
+    right: $fhi-space-2;
+}
+
+[class*="alert-fhi-"] {
```

Comment:
fhi- must always be a prefix

Reply:
In that case, we have to re-write the module